src/unittest/python/pmf_tests.py

```diff
@@ -91,6 +107,15 @@ def test_transform_results_in_predictions_on_transformed_values(self):
                                        'ANY_WIDTH')
         mock_log.assert_called_once_with('ANY_X_VALUE')
 
+    def test_deviance_test(self):
```

Comment:
Is test_deviance_test supposed to be defined twice? See line 84

Reply:
Agreed, but naming. This one is testing the "deviance_test". First `test_` is to signal that this is a unit test, then what is being tested. I'll change the name.